rusk-wallet/src/bin/command.rs

```diff
@@ -191,11 +191,11 @@ pub(crate) enum Command {
 
         /// Function name to call
         #[clap(short, long)]
-        fn_name: String,
+        name: String,
 
         /// Function arguments for this call
-        #[clap(short, long)]
-        fn_args: Vec<u8>,
+        #[clap(short = 'f', long)]
```

Comment:
I'm not familiar with clap, what does this do?

Reply:
There was a flag name conflict that's why I changed the names they should mean the same thing downstream